src/components/atoms/CustomIconMarker/index.js

```diff
@@ -0,0 +1,12 @@
+import markerIcon from '../../../images/marker-icon.png'
```

Comment:
you can use the resolves instead of `../../../images` 

like `_images/marker...`

Reply:
Do you know how can I call this? beacuse it's not working and probably I need to install something